src/css/styles.css

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+@import url("./reset.css");
+@import url("./variables.css");
+
+html,
+body {
+  font-family: var(--t-font-family);
+  font-size: var(--t-font-size);
+}
+
+h1,
+.is-h1 {
+  font-size: var(--t-font-size-xxlg);
+  line-height: var(--t-line-height-xxlg);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h2,
+.is-h2 {
+  font-size: var(--t-font-size-xlg);
+  line-height: var(--t-line-height-xlg);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h3,
+.is-h3 {
+  font-size: var(--t-font-size-lg);
+  line-height: var(--t-line-height-xlg);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h4,
+.is-h4 {
+  font-size: var(--t-font-size-md);
+  line-height: var(--t-line-height-md);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h5,
+.is-h5 {
+  font-size: var(--t-font-size-sm);
+  line-height: var(--t-line-height-sm);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h6,
+.is-h6 {
+  font-size: var(--t-font-size-xsm);
+  line-height: var(--t-line-height-xsm);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+p {
+  font-size: var(--t-font-size-md);
+  line-height: var(--t-line-height-md);
+  font-weight: var(--t-font-weight-regular);
+}
+
+/*
+ * Helpers
+ * ~ TOC ~
+ * 1.  Font Size
+ * 2.  Line Height
+ * 3.  Font Weight
+ */
+
+/* Font Size */
+
+.has-fs-xsm {
+  font-size: var(--t-font-size-xsm) !important;
+}
+.has-fs-sm {
+  font-size: var(--t-font-size-sm) !important;
+}
+.has-fs-md {
+  font-size: var(--t-font-size-md) !important;
+}
+.has-fs-lg {
+  font-size: var(--t-font-size-lg) !important;
+}
+.has-fs-xlg {
+  font-size: var(--t-font-size-xlg) !important;
+}
+.has-fs-xxlg {
+  font-size: var(--t-font-size-xxlg) !important;
+}
+
+/* Line Height */
+
+.has-lh-xsm {
+  line-height: var(--t-line-height-xsm) !important;
+}
+.has-lh-sm {
+  line-height: var(--t-line-height-sm) !important;
+}
+.has-lh-md {
+  line-height: var(--t-line-height-md) !important;
+}
+.has-lh-lg {
+  line-height: var(--t-line-height-lg) !important;
+}
+.has-lh-xlg {
+  line-height: var(--t-line-height-xlg) !important;
+}
```

Comment:
Whhy using important

Reply:
Used important because these are utility classes that are meant to overwrite any specific existing CSS property. This is a pattern I saw in bootstrap.